django_cron/backends/lock/database.py

```diff
@@ -5,13 +5,13 @@
 
 class DatabaseLock(DjangoCronJobLock):
     """
-    Locking cron jobs with database. Its good when you have not parallel run and want to make sure 2 jobs won't be
+    Locking cron jobs with database. It's good when you have not parallel run and want to make sure 2 jobs won't be
     fired at the same time - which may happened when job execution is longer that job interval.
     """
 
     @transaction.atomic
     def lock(self):
-        lock, created = CronJobLock.objects.get_or_create(job_name=self.job_name)
+        lock, created = CronJobLock.objects.select_for_update().get_or_create(job_name=self.job_name)
```

Comment:
Probably isn't needed for us, but good future proofing, resolves an issue listed [here](https://github.com/Tivix/django-cron/issues/243)